lib/service/service_test.go

```diff
@@ -1866,6 +1866,8 @@ func TestAgentRolloutController(t *testing.T) {
 	cfg.DebugService.Enabled = false
 	cfg.Auth.StorageConfig.Params["path"] = dataDir
 	cfg.Auth.ListenAddr = utils.NetAddr{AddrNetwork: "tcp", Addr: "127.0.0.1:0"}
+	// Speed up the reconciliation period from 1 minute to 200ms for testing purposes
```

Comment:
nit: Don't call out the values in case they change.

```suggestion
	// Speed up the reconciliation period for testing purposes.
```